### PR TITLE
Add notpad

### PR DESCRIPTION
**Pull request by Muhammed-Rahif**
This is the link to the project: Notpad ([Github](https://github.com/Muhammed-Rahif/Notpad), [Live](https://muhammed-rahif.github.io/Notpad/))

- [x] **I have read the [contributing guidelines](https://github.com/tauri-apps/awesome-tauri/blob/dev/.github/contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] If the project is closed source add the `![closed source]` badge after the `(link)` but before the `-`.
- [x] If the project/article is non-free or paywalled add the `![paid]` badge after the `(link)` but before the `-`.
- [x] Add entries alphabetically to the most appropriate category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [ ] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

**Comment by aurorarissime** [comment]
You named it "notpad" in your file changes but "notepad" everywhere else?

**Comment by Muhammed-Rahif** [comment]
My bad! Its actually "Notpad".

File: README.md [hunk 1]
```diff
@@ -298,6 +298,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [MDX Notes](https://github.com/maqi1520/mdx-notes/tree/tauri-app) - Versatile WeChat typesetting editor and cross-platform Markdown note-taking software.
 - [Noor](https://noor.to/) ![closed source] - Chat app for high-performance teams. Designed for uninterrupted deep work and rapid collaboration.
 - [Notedown](https://github.com/ruralad/notedown) - Minimal, local-first cross-platform note taking app.
+- [Notpad](https://github.com/Muhammed-Rahif/Notpad) - The ordinary notepad that everyone know, but it's rich!
```
Comment (FabianLars) [inline README.md]:
```suggestion
- [Notpad](https://github.com/Muhammed-Rahif/Notpad) - The ordinary notepad that everyone knows, but it's rich!
```

Would you mind a more, i don't know, "objective" description? The other apps are described more "flatly" and i'd prefer to keep it in line tbh. Also, i think not everyone understands what "it's rich!" actually entails?

Comment (Muhammed-Rahif) [reply README.md]: What about "Cross-platform rich text editor with a notepad interface, enhanced with advanced features beyond standard notepad."

Comment (FabianLars) [reply README.md]: yeah, that would work for me :) thanks

Comment (Muhammed-Rahif) [reply README.md]: Let me change.

**Review by FabianLars — APPROVED** [review]

**Pull request merged**